### PR TITLE
3rd Pull Request for 3rd Microverse Project

### PR DESCRIPTION
**Pull request by nrqrmz**
Fix all changes required by TSE in the last pull request.

### CODE REVIEW

**Review by elmejdki — CHANGES_REQUESTED** [review]
# status: require changes :recycle: 
## great job guys so far but in order to make this project approved you need to make some changes :hammer: :toolbox: :hammer_and_wrench: 

# :warning: never close a pull request that was not approved, you should use the same reviewed pull request in the next code review request.

- [x] the buttons should go to the rightmost part of the navbar you can decrease there container width in order to correct their position:
image


- [x] add box-shadow or border-bottom to the navbar to make it stand out from the main container:
### clone:
image
### original:
image

- [x] Fix the size of the video make it a little bit higher:
image

- [x] Give some padding-left to this section to separate it from the other one:
image

- [x] Use some padding here to separate the icon from the text:
image

- [x] the comments buttons the social media icons should go in the same row please fix that:
### clone:
image


### original:
image

### Please make all the required changes before you request another code review.
## Happy Coding :tophat:

File: index.html [hunk 1]
```diff
@@ -0,0 +1,641 @@
...
+      <div id="thcont">
+        <div id="thanks" class="center-items">          
+          <p class="bold">Thanks for reading The Times.</p>
+          <a href="https://www.nytimes.com/subscription?campaignId=7UU8U" class="bold">Subscribe to The Times</a>         
+        </div>
+      </div>
+    </div>
+    <aside id="asidebox">
+      <div id="asideboxmt">
+        <div id="asideitems">
```
Comment (elmejdki) [inline index.html:106]: please changes the id's to classes you are using them a lot, it's not a good practice at all.

**Comment by nrqrmz** [comment]
Add all required changes.

**Review by andresporras3423 — CHANGES_REQUESTED** [review]
**Hello @nrqrmz** 
Good job with your new submission for this project :muscle:  
Your project passes most of the requirements :clap: :clap: :clap:  
 There are still some details to improve before you can go ahead:
- [x]  The grid of your page looks like this (1200px wide screen):



Add some padding between the paragraph of your grid and make sure the text doesn't overlap the background draws.

- [x]  add left padding in most popular section:



- [x] After the end of the article, the "read comments" button and the icons should be in the same line. Your page:



Please request another review once you are ready! :+1: 

Happy coding :relaxed:

Oscar Andrés Russi Porras

**Comment by nrqrmz** [comment]
### Fix all the required changes:

- Add space between most popular column and the images grid

- Fix overlapping issue relating paragraphs and inflation image

- Fix linters and social buttons at the end of the article

**Review by HADDADSOHAIB — APPROVED** [review]
### Status: Project Approved :+1: 

### Hi @nrqrmz  :hand: 

### You did great with the project, you implemented all changes required with the previous TSEs, you can go on and merge this branch into master.However, there is some [Optional] changes that you can do before

- [Optional] Use the `<main>` tag to wrap the main part of the page.

**Pull request merged**